webapp/views.py

```diff
@@ -151,6 +172,9 @@ def get_context_data(self, **kwargs):
         context['items'] = self._get_resources()
         context['topic_slug'] = self.request.GET.get('topic')
         context['content_slug'] = self.request.GET.get('content')
+        context['search_slug'] = ''
+        if (self.request.GET.get('search')):
```

Comment:
Bind this to a variable so you don't call the `GET.get('search')` twice (if that's a thing in Python)?